angular/src/components/settings/vault-timeout-input.component.ts

```diff
@@ -18,7 +18,18 @@ export class VaultTimeoutInputComponent implements ControlValueAccessor, Validat
     return this.form.get("vaultTimeout").value === VaultTimeoutInputComponent.CUSTOM_VALUE;
   }
 
+  get customTimeInMinutes() {
+    return this.form.get("custom.hours")?.value * 60 + this.form.get("custom.minutes")?.value;
+  }
+
+  get exceedsMinimumTimout(): boolean {
+    return (
+      !this.showCustom || this.customTimeInMinutes > VaultTimeoutInputComponent.MIN_CUSTOM_MINUTES
+    );
+  }
```

Comment:
Should we make these private? They are only used internally right?

Reply:
`exceedsMinimumTimout` is used in a view but I would make `customTimeInMinutes` private